### DISCUSSION TITLE
Explore other Xarray climate science packages

### DISCUSSION THREAD

**Discussion by tomvothecoder** [xCDAT/xcdat#498] [3. Ideas]
The Xarray ecosystem includes a ton geosciences packages listed here: https://docs.xarray.dev/en/stable/ecosystem.html.
It is a good idea to understand their capabilities and refer xCDAT users to them where it makes sense.

Examples:
- [xmip](https://cmip6-preprocessing.readthedocs.io/en/latest/api.html)
- [xclim](https://xclim.readthedocs.io/en/stable/)
- [geocat-comp](https://geocat-comp.readthedocs.io/en/latest/index.html) -- this package has some similar functionalities (climatologies, anomalies, interpolation). We should investigate the differences such as what one package does better over the other, edge cases that are handled, etc.
- [xoa](https://xoa.readthedocs.io/en/latest/index.html)
  - Related GH [discussions post](https://xoa.readthedocs.io/en/latest/index.html)

**Comment by jypeter** [comment]
Do not forget to check xoa (#295) as well, even if it is not listed (yet) in the `xarray` documentation link listed above. 

@stefraynaud, you may want to add your package to https://docs.xarray.dev/en/stable/ecosystem.html

**Reply by tomvothecoder** [reply]
Thank you @jypeter! I added `xoa` to the examples list above.
